Remove ts-node usage for matrix unittest and align nyc config with the rest of the repo

curtisman opened the pull request:
Also:
- remove starting tinylicous on test:mocha script.   It is only needed for realsvc tests.
- align build scripts for driver-utils packages

curtisman committed:
Remove ts-node usage for matrix unittest and align nyc config with the rest of the repo

tylerbutler commented on packages/dds/matrix/src/test/matrix.big.spec.ts:
```diff
@@ -187,8 +185,8 @@ describe("Big Matrix", function () {
         // lastSequenceNumber to be updated.
-        // Sp, we test summarize with local client because MockFluidDataStoreRuntime has no delta manager and is assigned
-        // one once it is connected.
+        // Sp, we test summarize with local client because MockFluidDataStoreRuntime has no delta manager and is
```
```suggestion
        // So, we test summarize with local client because MockFluidDataStoreRuntime has no delta manager and is
```

tylerbutler reviewed (approved):
:shipit: (once CI passes 😄 )

DLehenbauer reviewed (approved):
Thanks for bringing me into compliance!

DLehenbauer committed:
Create matrix.stress.spec.ts

Lint found a cosmetic bug: the iteration count should increment each time we complete a round.

Feel free to include or let me fix up after.

DLehenbauer commented on packages/dds/matrix/src/test/matrix.stress.spec.ts:
```diff
@@ -263,13 +271,14 @@ describe("Matrix", () => {
-                async function () {
-                    this.timeout(15000);
+                async function() {
+                    this.timeout(20000);
```
Do you need a faster PC?  :-)

curtisman replied:
I start hitting it with coverage builds.  I guess it wasn't ran with coverage before?

curtisman committed:
Update packages/dds/matrix/src/test/matrix.big.spec.ts

Co-authored-by: Tyler Butler <tyler@tylerbutler.com>
